README.md

```diff
@@ -28,14 +28,222 @@ In addition, that format would include optional **signing** of the resources, wh
 
-There is already a [packaging format proposal](https://w3ctag.github.io/packaging-on-the-web/) which we will base upon.
-We are proposing to improve on the spec, in particular by introducing 3 major additions:
+This is roughly based on the existing [Packaging on the Web](https://w3ctag.github.io/packaging-on-the-web/) W3C TAG proposal, but we've made significant changes:
 
-1. Hierarchical structure of the sub-packages to allow resources from multiple origins.
-2. Index of content to facilitate local resource fetching from the package.
+1. Index of content to facilitate local resource fetching from the package.
+2. Package manifests that can refer to sub-package manifests to allow resources
+   from multiple origins.
 3. Signature block, to cryptographically sign the content of the package.
+4. Removed the
+   [fragment-based URL schema](https://w3ctag.github.io/packaging-on-the-web/#fragment-identifiers)
+   from the spec as it's complex with limited use cases.
+
+Note that this is just an explainer, **not a specification**. We'll add more
+precision when we translate it to a spec.
+
+### Overall format
+
+The package is structured as follows. All "offsets" are 8-byte little-endian
+integers representing a byte offset from the start of the initial magic number.
+
+1. A magic number: `F0 9F 8C 90 F0 9F 93 A6` (🌐📦 in UTF-8).
+2. The length of the package (all bytes from the first of the initial magic
+   number to the last of the final magic number) as a little-endian 8-byte
+   integer.
+3. A 1-byte count of metadata sections followed by that many of the following
+   entries. Repeated entry IDs and non-monotonically-increasing offsets cause
+   the package to fail to parse.
+   * The byte `01` indicating an [index](#index), followed by the offset of the
+     start of the index.
+   * The byte `02` indicating a block of [certificates](#certificates), followed
+     by the offset of the start of the certificates.
+   * The byte `03` indicating a [manifest](#manifest), followed by the offset of
+     the start of the manifest.
+
+   Unknown entry IDs are assumed to be followed by an offset, and the pair must
+   be ignored.
+4. The offset of the [main content](#main-content) of the package.
+5. The metadata sections promised above.
+6. A sequence of HTTP responses constituting the main content of the package.
+   Mismatches with the manifest usually cause the package to fail to parse, as
+   described in the [manifest](#manifest) section, but mismatches with the index
+   might not.
+7. The length of the package, repeated from earlier. This allows tools to
+   operate on the package even if it's been prepended with a self-extracting
+   executable.
+8. The same magic number: `F0 9F 8C 90 F0 9F 93 A6` (🌐📦 in UTF-8).
+
+The MIME type of this format is `application/package`. TODO: Is that right?
```

Comment:
I suspect that "package" is too general to suggest, and that we should suggest something more specific like `webpackage` or `httppackage` (since it holds HTTP responses).